.storybook/webpack.config.js

```diff
@@ -9,27 +10,32 @@ module.exports = (storybookBaseConfig, configType) => {
   // You can change the configuration based on that.
   // 'PRODUCTION' is used when building the static version of storybook.
 
-  // TODO: I would like to remove this rule, and let load CSS via default config
-  // But unless I specify this rule, eslint-loader tries to lint our CSS, failing.
-  // .eslintignore doesn't work neither
+  storybookBaseConfig.plugins.push(
+      new StyleLintPlugin({
+          configFile: path.resolve(__dirname, '../.stylelintrc.js'),
+          context: path.resolve(__dirname, '../src'),
+          syntax: 'scss'
+      }),
+  );
+
   storybookBaseConfig.module.rules.push({
-    test: /\.css$/,
+    test: /\.scss$/,
     loaders: [
-      "style-loader", 
+      "style-loader",
       {
         loader: "css-loader",
         options: {
-          importLoaders: 1,
+            importLoaders: 1,
         }
-      }
+      },
+      "sass-loader"
     ],
-    include: path.resolve(__dirname, '../')
+    include: path.resolve(__dirname, '../src')
```

Comment:
I wonder if it's worth it to add also postcss with autoprefixer,  like the default config of storybook's wepback does https://github.com/storybooks/storybook/blob/master/app/react/src/server/config/defaults/webpack.config.js
(then importLoaders should be 2)
Anyway, this can be in a separate PR

Reply:
Good idea, I'll look to do that in a future PR  👍🏼